docs/book/src/cronjob-tutorial/testdata/project/dist/chart/templates/metrics/metrics-service.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "chart.labels" . | nindent 4 }}
+    control-plane: controller-manager
```

Comment:
I think we need to add to the ServiceMonitor as well, see:

https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-with-plugins/config/prometheus/monitor.yaml#L3-L6C2

https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-with-plugins/dist/chart/templates/prometheus/monitor.yaml

Could you please ensure that we have the label in the same places as we have for kustomize?